book/src/proposals/booting-from-snapshot.md

```diff
@@ -0,0 +1,218 @@
+# Booting From a Snapshot
+
+## Problem
+
+A validator booting from a snapshot should confirm that they are caught up to
+the correct cluster before voting. There are two major risks if they do not:
+
+### Wrong Cluster
+
+The validator may be fed a snapshot and gossip info for a different (though not
+necessarily malicious) cluster. If that cluster is at slot 100,000 while the
+cluster the validator intended to join is at 10,000, if the validator votes
+once on the wrong cluster, it is locked out of the correct cluster for 90,000
+slots.
```

Comment:
> it is locked out of the correct cluster for 90,000 slots.

It would be nice to explain why this is the case.

Reply:
Updated. look good?